<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>678</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>archpm</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="pagecnt">
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="next">
         <property name="font">
          <font>
           <family>Noto Sans Thai</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prev">
         <property name="font">
          <font>
           <family>Noto Sans Thai</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insall">
         <property name="font">
          <font>
           <family>Noto Sans Thai</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>InsAll</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rmall">
         <property name="font">
          <font>
           <family>Noto Sans Thai</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>RmAll</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="installed">
         <property name="font">
          <font>
           <family>Noto Sans Thai UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Installed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="orphans">
         <property name="font">
          <font>
           <family>Noto Sans Thai UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Orphans</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updates">
         <property name="font">
          <font>
           <family>Noto Sans Thai UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Updates</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="depend">
         <property name="font">
          <font>
           <family>Noto Sans Thai UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Depend</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="method">
            <item>
             <property name="text">
              <string>Internal Regex</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>KMP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Trie</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="boxTreename">
            <property name="font">
             <font>
              <family>Noto Sans Thai UI</family>
             </font>
            </property>
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>core</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>community</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>extra</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>local</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editorSearch"/>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSearch">
            <property name="font">
             <font>
              <family>Noto Sans Thai UI</family>
             </font>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>3</number>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>866</width>
               <height>22</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="entries_display">
                <property name="sizeConstraint">
                 <enum>QLayout::SetNoConstraint</enum>
                </property>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionDatabase_Update"/>
    <addaction name="separator"/>
    <addaction name="actionFixlck"/>
    <addaction name="separator"/>
    <addaction name="actionCleandis"/>
    <addaction name="actionClean_cache"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Author"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Thai UI</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionInstallation">
   <property name="text">
    <string>Installed Packages</string>
   </property>
  </action>
  <action name="actionRemove_Package">
   <property name="text">
    <string>Remove Packages</string>
   </property>
  </action>
  <action name="actionSoftware_Source">
   <property name="text">
    <string>Software Source</string>
   </property>
  </action>
  <action name="actionFixlck">
   <property name="text">
    <string>Fix DBLock</string>
   </property>
  </action>
  <action name="actionAbout_Author">
   <property name="text">
    <string>About Archpm</string>
   </property>
  </action>
  <action name="actionAbout_Author_2">
   <property name="text">
    <string>About Author</string>
   </property>
  </action>
  <action name="actionUser_Guider">
   <property name="text">
    <string>User Guider</string>
   </property>
  </action>
  <action name="actionInstall_from_file">
   <property name="text">
    <string>Lauch Terminal</string>
   </property>
  </action>
  <action name="actionCleandis">
   <property name="text">
    <string>Clean Display</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionDatabase_Update">
   <property name="text">
    <string>Database Update</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Back Up</string>
   </property>
  </action>
  <action name="actionClean_cache">
   <property name="text">
    <string>Clean Cache</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
